FilmFestivalLoader/IDFA/parse_idfa_html.py

```diff
@@ -9,209 +9,149 @@
 @author: maarten
 """
 
+import datetime
 import os
-import sys
 import re
-import datetime
-import html.parser
+from enum import Enum, auto
+from typing import Dict
 
-prj_dir = os.path.expanduser("~/Projects/FilmFestivalPlanner")
-shared_dir = os.path.join(prj_dir, "FilmFestivalLoader/Shared")
-sys.path.insert(0, shared_dir)
-import planner_interface as planner
-import application_tools as app_tools
-import web_tools
+import Shared.planner_interface as planner
+from Shared.application_tools import ErrorCollector, DebugRecorder, comment
+from Shared.parse_tools import FileKeeper, HtmlPageParser, ScreeningKey, try_parse_festival_sites
+from Shared.web_tools import UrlFile, iri_slug_to_url
 
 # Parameters.
 festival = 'IDFA'
 festival_city = 'Amsterdam'
-festival_year = 2020
-az_page_count = 27
-
-# Directories.
-project_dir = os.path.expanduser(f"~/Documents/Film/{festival}/{festival}{festival_year}")
-webdata_dir = os.path.join(project_dir, "_website_data")
-plandata_dir = os.path.join(project_dir, "_planner_data")
-
-# Filename formats.
-az_file_format = os.path.join(webdata_dir, "azpage_{:02d}.html")
-film_file_format = os.path.join(webdata_dir, "filmpage_{:03d}.html")
+festival_year = 2022
+az_page_count = 30
 
 # Files.
-filmdata_file = os.path.join(plandata_dir, "filmdata.csv")
-filminfo_file = os.path.join(plandata_dir, "filminfo.xml")
-debug_file = os.path.join(plandata_dir, "debug.txt")
+fileKeeper = FileKeeper(festival, festival_year)
+debug_file = fileKeeper.debug_file
+
+plandata_dir = fileKeeper.plandata_dir
+filmdata_file = os.path.join(plandata_dir, 'filmdata.csv')
+filminfo_file = os.path.join(plandata_dir, 'filminfo.xml')
```

Comment:
These files (`.csv`, `.xml`) don't belong in `FileKeeper` but in `planner_interface`.

Removed the lined, because both `filmdata_file ` and `filminfo_file ` are unused.